packages/framework/src/Resources/config/services.yaml

```diff
@@ -1125,3 +1125,5 @@ services:
 
     twig.stringloader:
         class: Twig\Loader\ArrayLoader
+
+    Shopsys\FrameworkBundle\Model\Watchdog\Mail\WatchdogMail: ~
```

Comment:
You should run ./phing yaml-standards-fix to fix the alphabetical order of this file